packages/lodestar/src/network/interface.ts

```diff
@@ -4,12 +4,22 @@
 import PeerInfo from "peer-info";
 import {EventEmitter} from "events";
 
-import {
-  Attestation, BeaconBlock, Shard, ResponseBody, RequestBody,
-} from "../types";
-import {RequestId, Method} from "../constants";
+import {Attestation, BeaconBlock, RequestBody, ResponseBody, Shard,} from "../types";
+import {ATTESTATION_TOPIC, BLOCK_TOPIC, Method, RequestId} from "../constants";
+import StrictEventEmitter from "strict-event-emitter-types";
 
-export interface INetwork extends EventEmitter {
+interface INetworkEvents {
+  [BLOCK_TOPIC]: (block: BeaconBlock) => void;
+  [ATTESTATION_TOPIC]: (attestation: Attestation) => void;
+  ["gossipsub:heartbeat"]: void;
```

Comment:
Maybe we can make all of these constants

Gossip.heartbeat?

Reply:
I thinks thats how is defined in spec. Anyways, I didn't though logic haha